etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -190,7 +157,7 @@ stackhpc_controller_firewalld_rules_template:
   # Octavia
   - rules:
       - port: 5555/udp
-        network: "{{ octavia_net_name }}"
+        network: "{{ octavia_net_name | default(public_net_name) }}"
```

Comment:
I think let's remove the default and make the `enabled` flag dependent upon `octavia_net_name` being defined. If you're only using the OVN backend you don't need this network.

Reply:
It doesn't like not having any default at all for ```octavia_net_name``` so I've made it blank. Otherwise, done